README.md

```diff
@@ -3,23 +3,26 @@
 [![DOI](https://zenodo.org/badge/632689876.svg)](https://zenodo.org/badge/latestdoi/632689876)
 [![Chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://acquire-imaging.zulipchat.com/)
 
-# acquire-imaging
+# Acquire (`acquire-imaging` on PyPI)
 
-This python package provides a multi-camera video streaming library focusing
-on performant microscopy.
+Acquire ([`acquire-imaging` on PyPI](https://pypi.org/project/acquire-imaging/)) provides high-speed, multi-camera, video streaming for up to **2** cameras and image acquisition with a programming interface for streaming video data directly to Python, cloud-friendly file formats, and visualization platforms, such as [napari](https://napari.org/stable/).
 
 > **Note** This is an early stage project. If you find it interesting please
 > reach out!
 
-Support for:
+Acquire supports the following cameras (currently only on Windows):
 
-- Up to two independent video streams
-- Camera support:
-  - Hamamatsu Orca Fusion BT (C15440-20UP) (windows only)
-  - Vieworks VC-151MX-M6H00
-- Streaming file format support:
-  - Tiff
-  - Zarr v2
+- [Hamamatsu Orca Fusion BT (C15440-20UP)](https://www.hamamatsu.com/eu/en/product/cameras/cmos-cameras/C15440-20UP.html)
+- [Vieworks VC-151MX-M6H00](https://www.visionsystech.com/products/cameras/vieworks-vc-151mx-sony-imx411-sensor-ultra-high-resolution-cmos-camera-151-mp)
+- [FLIR Blackfly USB3 (BFLY-U3-23S6M-C)](https://www.flir.com/products/blackfly-usb3/?model=BFLY-U3-23S6M-C&vertical=machine+vision&segment=iis)
+- [FLIR Oryx 10GigE (ORX-10GS-51S5M-C)](https://www.flir.com/products/oryx-10gige/?model=ORX-10GS-51S5M-C&vertical=machine+vision&segment=iis)
+
+Acquire also supports the following output file formats:
+
+- [Tiff](https://en.wikipedia.org/wiki/TIFF)
+- [Zarr](https://zarr.dev/)
+
+For testing and demonstration purposes, Acquire provides a few simulated cameras, as well as raw and trash output devices.
```

Comment:
```suggestion
For testing and demonstration purposes, Acquire provides a few simulated video sources.
```

This might be clearer if we don't mention the raw/trash output devices. They're there for functional reasons, not just for test and demonstration.

"Raw" is kind of a file format in its own right. "Trash" has to be there in case someone does not want to write data to disk. I don't think these need to be mentioned in the README, though they should be documented.